mon-pix/tests/unit/routes/authentication/login-oidc-test.js

```diff
@@ -95,30 +95,59 @@ module('Unit | Route | login-oidc', function (hooks) {
           assert.strictEqual(sessionStub.data.nextURL, '/organisations/PIXOIDC01/acces');
         });
 
-        test('should build the url from the intent name and contexts in session data nextUrl', async function (assert) {
-          // given
-          const authenticateStub = sinon.stub().resolves();
-          const sessionStub = Service.create({
-            attemptedTransition: { intent: { name: 'organizations.access', contexts: ['PIXOIDC01'] } },
-            authenticate: authenticateStub,
-            data: {},
+        module('when there is at least one context', function () {
+          test('should build the url from the intent name and contexts in session data nextUrl', async function (assert) {
+            // given
+            const authenticateStub = sinon.stub().resolves();
+            const sessionStub = Service.create({
+              attemptedTransition: { intent: { name: 'organizations.access', contexts: ['PIXOIDC01'] } },
+              authenticate: authenticateStub,
+              data: {},
+            });
+            const route = this.owner.lookup('route:authentication/login-oidc');
+            route.set('session', sessionStub);
+            sinon.stub(Location, 'assign');
+            route.router.urlFor = sinon.stub();
+
+            const transition = {
+              to: { queryParams: {}, params: { identity_provider_slug: 'oidc-partner' } },
+              abort: sinon.stub(),
+            };
+
+            // when
+            await route.beforeModel(transition);
+
+            // then
+            sinon.assert.calledWith(route.router.urlFor, 'organizations.access', 'PIXOIDC01');
+            assert.ok(true);
+          });
+        });
+        module('when there is no context', function () {
```

Comment:
Sauter 1 ligne ici : 

```suggestion
        });

        module('when there is no context', function () {
```